src/basic-syntax/rustdoc.md

```diff
@@ -6,6 +6,13 @@ All language items in Rust can be documented using special `///` syntax.
 /// Determine whether the first argument is divisible by the second argument.
 ///
 /// If the second argument is zero, the result is false.
+///
+/// # Example
+/// ```
+/// assert!(is_divisible_by(42, 2));
+/// assert!(!is_divisible_by(5, 3));
+/// assert!(!is_divisible_by(13, 0));
```

Comment:
Showing off a doctest is a great idea! However, I'm always afraid of losing vertical space, so let's just show it with a single line:

```suggestion
```